builders/build/substrate-api/chopsticks.md

```diff
@@ -153,23 +152,26 @@ npx @acala-network/chopsticks run-block --endpoint wss://wss.api.moonbeam.networ
 To test out XCM messages between networks, you can fork multiple parachains and a relay chain locally. For example, the following will fork Moonriver, Karura, and Kusama given that you've downloaded the [config folder from the GitHub repository](https://github.com/AcalaNetwork/chopsticks/tree/master/configs){target=_blank}:  
 
 ```
-npx @acala-network/chopsticks xcm --relaychain=configs/kusama.yml --parachain=configs/moonriver.yml --parachain=configs/karura.yml
+npx @acala-network/chopsticks xcm --r=kusama.yml --p=moonriver.yml --p=karura.yml
```

Comment:
```suggestion
npx @acala-network/chopsticks xcm \
--r=kusama.yml \
--p=moonriver.yml \
--p=karura.yml
```

Reply:
maybe add a note here stating that the chain name and raw github url can also be used